internal/app/api/v1/scripts.go

```diff
@@ -173,21 +175,34 @@ func (s kubtestAPI) ExecuteScript() fiber.Handler {
 			execution,
 			request.Params,
 		)
-		s.Repository.Insert(ctx, scriptExecution)
+		err = s.Repository.Insert(ctx, scriptExecution)
+		if err != nil {
+			return s.Error(c, http.StatusBadGateway, fmt.Errorf("inserting script CR error: %w", err))
 
+		}
+		fmt.Println("*******ID:", scriptExecution.Id)
+		jobClient, err := jobs.NewJobClient()
+		if err != nil {
+			return s.Error(c, http.StatusInternalServerError, fmt.Errorf("can't get k8s client: %w", err))
+		}
 		// save watch result asynchronously
-		go func(scriptExecution kubtest.ScriptExecution, executor client.HTTPExecutorClient) {
+		go func(scriptExecution kubtest.ScriptExecution) error {
 			// watch for execution results
-			execution, err = executor.Watch(scriptExecution.Execution.Id, func(e kubtest.Execution) error {
 
-				l := s.Log.With("executionID", e.Id, "status", e.Status, "duration", e.Duration().String())
-				l.Infow("saving", "result", e.Result)
-				l.Debugw("saving - debug", "scriptExecution", scriptExecution)
+			// l := s.Log.With("executionID", e.Id, "status", e.Status, "duration", e.Duration().String())
+			// l.Infow("saving", "result", e.Result)
+			// l.Debugw("saving - debug", "scriptExecution", scriptExecution)
 
-				scriptExecution.Execution = &e
-				return s.Repository.Update(ctx, scriptExecution)
-			})
-		}(scriptExecution, executor)
+			result := jobClient.LaunchK8sJob(scriptExecution.Id, getImageFromCRType(scriptCR.Spec.Type_), options)
```

Comment:
just thinking if we could go async here 
- we have pending `ABORT` feature - which will need to be implemented - probably we could run this job and keep some ID of it to eventually kill execution.

Reply:
job's name is the same as execution ID so it shouldn't be a problem to do that